src/stores/contactmoment/index.ts

```diff
@@ -35,6 +37,18 @@ export const useContactmomentStore = defineStore("contactmoment", {
     stop() {
       this.$reset();
     },
+    addZaak(zaak: Zaak) {
+      const contactmomentZaak = zaak as ContactmomentZaak;
+      const index = this.zaken.findIndex((element) => element.id === zaak.id);
```

Comment:
I'd opt for `((element)) => ((_zaak))` to make it more descriptive.

Reply:
noted. nemen we mee met een volgende aanpassing aan het contactmoment